boards/nordic/nrf54h20dk/board.yml

```diff
@@ -0,0 +1,8 @@
+board:
+  name: nrf54h20dk
+  vendor: nordic
+  socs:
+  - name: nrf54h20
+    variants:
+    - name: ram
+      cpucluster: cpuppr
```

Comment:
as said: change this to `xip`, also, adjust the snippets so that we have a `nordic-ppr-xip`

Reply:
done